python/semantic_kernel/connectors/ai/open_ai/services/open_ai_handler.py

```diff
@@ -96,19 +155,3 @@ def _handle_structured_output(
             # Case 3: response_format is a dictionary, pass it without modification
             elif isinstance(response_format, dict):
                 settings["response_format"] = response_format
-
-    def store_usage(
-        self,
-        response: ChatCompletion
-        | Completion
-        | AsyncStream[ChatCompletionChunk]
-        | AsyncStream[Completion]
-        | CreateEmbeddingResponse,
-    ):
-        """Store the usage information from the response."""
-        if not isinstance(response, AsyncStream) and response.usage:
-            logger.info(f"OpenAI usage: {response.usage}")
-            self.prompt_tokens += response.usage.prompt_tokens
```

Comment:
no longer storing this is also breaking, not sure how many people use it, but let's discuss!

Reply:
Sounds good, we should discuss!

This is specific to OpenAI and Azure OpenAI. We are not storing usage in any other connectors. And I don't think it's a valid pattern to store usage at the connector level. We provide usage in each response and users can track usage with their own logic.

In addition, OpenAI has introduced other usage types, such as `reasoning_tokens`, which we are not considering. So, I think the best solution for us to go forward is to remove this completely.

Reply:
I do agree that we shouldn't remove this. We don't know who is depending on the data.

Reply:
Your argument makes sense, but I still don't think we should remove it since it's a breaking change. I do think it would make sense to deprecate usage of this and specify what should be used instead.